Source/WebCore/dom/Node.cpp

```diff
@@ -2506,7 +2510,9 @@ bool Node::willRespondToMouseMoveEvents() const
         return false;
 #endif
     auto& eventNames = WebCore::eventNames();
-    return hasEventListeners(eventNames.mousemoveEvent) || hasEventListeners(eventNames.mouseoverEvent) || hasEventListeners(eventNames.mouseoutEvent);
+    return eventTypes().containsIf([&](const auto& type) {
```

Comment:
auto should suffice. If I understand correctly, this resolves to an enum type so no need to pass by const ref.

Reply:
It's an AtomString